src/main/java/org/zalando/nakadi/service/subscription/zk/AbstractZkSubscriptionClient.java

```diff
@@ -102,7 +103,7 @@ public final void runLocked(final Runnable function) {
             if (releaseException != null) {
                 throw releaseException;
             }
-        } catch (final NakadiRuntimeException e) {
+        } catch (final NakadiRuntimeException | MyNakadiRuntimeException1 e) {
```

Comment:
Maybe it is better not to mix to runtime exception root classes and use only one of them?